src/xscen/biasadjust.py

```diff
@@ -60,8 +60,8 @@ def train(
     period: list[str],
     *,
     method: str = "DetrendedQuantileMapping",
-    group: sdba.Grouper | str | dict | None = None,
-    xclim_train_args: dict | None = None,
+    group: xsdba.Grouper | str | dict | None = None,
+    xsdba_train_args: dict | None = None,
```

Comment:
This will be a breaking change. 

We should instead support both options for a while, but issue a `FutureWarning` if someone uses `xclim_train/adjust_args`. The rest of the code can use the new names, just with `xsdba_train/adjust_args = xclim_train/adjust_args` for those cases.